src/pages/ServerRoleSummary/EditCloudSettings.js

```diff
@@ -35,38 +35,70 @@ class EditCloudSettings extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      key: TAB.NIC_MAPPINGS
+      key: TAB.NIC_MAPPINGS,
+      showCloseConfirmation: false
     };
+    this.dataChanged = [false, false, false, false, false];
```

Comment:
This is a little bit painful, as we'll have to remember to expand it if we add additional tabs

Reply:
I'm all ears if you have a better idea.